tests/jest/unit/app/modules/import-export/assets/js/admin.test.js

```diff
@@ -0,0 +1,121 @@
+describe( 'Import Export Admin Test', () => {
+	window.elementorAppConfig = {
+		'import-export': {
+			lastImportedSession: {
+				kit_title: 'Kit Title',
+			},
+		},
+	};
+
+	require( 'elementor/app/modules/import-export/assets/js/admin.js' );
+
+	it( 'should check if "kit deleted" dialog is not shown without cache', () => {
+		// Arrange
+		window.elementorCommon = {
+			dialogsManager: {
+				createWidget: jest.fn( () => {
+					return {
+						show: jest.fn( () => {} ),
+					};
+				} ),
+			},
+		};
+
+		// Act
+		window.dispatchEvent( new Event( 'load' ) );
+
+		// No Kit referrer dialog shown without refferer in cache
+		expect( window.elementorCommon.dialogsManager.createWidget ).not.toHaveBeenCalled();
+	} );
+
+	it( 'should check if the referred "kit deleted" dialog is shown', () => {
+		// Arrange
+		window.elementorCommon = {
+			dialogsManager: {
+				createWidget: jest.fn( ( type, options ) => {
+					// Auto confirm
+					sessionStorage.setItem( 'elementor-kit-data', JSON.stringify( {
+						referrerKitId: '123',
+						activeKitName: 'Active Kit Name',
+					} ) );
+
+					// To expect
+					window.elementorCommon.lastWidgetHeaderMessage = options.headerMessage;
+					window.elementorCommon.lastWidgetMessage = options.message;
```

Comment:
you can use `.toHaveBeenCalledWith()`

Reply:
I tried this. In this case the second `options` parameter is an object and not possible to check equality with it